cmd/queue.go

```diff
@@ -49,6 +52,20 @@ func (i *Item) List() ([]resources.Resource, error) {
 	return listers[i.Type](i.Region.Session)
 }
 
+func (i *Item) GetProperty(key string) string {
+	if key == "" {
+		return i.Resource.String()
+	}
+
+	getter, ok := i.Resource.(resources.ResourcePropertyGetter)
+	if !ok {
+		logrus.Warnf("%T does not support custom properties", i.Resource)
```

Comment:
IMHO we should fail if the user provided config is incorrect

Reply:
In this case I have to see how to bubble this error up, since we are in a goroutine. The only way that currently comes into my mind is an ugly error channel.

Reply:
Will do in a separate PR to keep the history clean: https://github.com/rebuy-de/aws-nuke/pull/200